### PR TITLE
GoTypeBear.py: Update the package reference

### PR DESCRIPTION
**Pull request by Vamshi99**
Change the old package to new one, as it is removed
and updated to a new one.

Closes https://github.com/coala/coala-bears/issues/1488

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: bears/go/GoTypeBear.py [hunk 1]
```diff
@@ -23,4 +23,4 @@ class GoTypeBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '-e', filename
+        return ('-e', filename)
```
Comment (nemaniarjun) [inline bears/go/GoTypeBear.py]: Why did you make this change?

Comment (Vamshi99) [inline bears/go/GoTypeBear.py]: I was getting error in this line, so was just checking if anything changes by this. Though, it doesn't affect the code.

**Comment by jayvdb** [comment]
The approach of using https://github.com/golang/go 's gotype will fail because our CI uses a version of Go not supported by the golang repo.
`go/src/go/types/gotype.go:279: undefined: types.SizesFor`

**Comment by jayvdb** [comment]
ack a69410f

**Comment by jayvdb** [comment]
reack 17aa6d5

**Comment by jayvdb** [comment]
reack 1fa143c

**Comment by jayvdb** [comment]
@rultor merge

**Comment by jayvdb** [comment]
ack 1fa143c

**Pull request merged**